connaisseur/flask_application.py

```diff
@@ -116,14 +97,56 @@ def readyz():
     return "", 200
 
 
+async def __async_mutate():
+    # Maximum timeout for admission control is 30s
+    # If Connaisseur issues requests that timeout after the webhook times out
+    # k8s will report that Connaisseur was unresponsive, which isn't quite true
+    # thus we timeout slightly earlier
+    # https://github.com/sse-secure-systems/connaisseur/issues/448
+    timeout = aiohttp.ClientTimeout(total=const.AIO_TIMEOUT_SECONDS)
+    async with aiohttp.ClientSession(timeout=timeout) as session:
+        admission_request = None
+        try:
+            logging.debug(request.json)
+            admission_request = AdmissionRequest(request.json)
+            response = await __admit(admission_request, session)
+            # Depending on whether alerting must succeed, run it synchronously or in the background
+            dispatch_alerts(admission_request, True)
+            return jsonify(response)
+        except Exception as err:
+            if isinstance(err, BaseConnaisseurException):
+                err_log = str(err)
+                msg = err.user_msg  # pylint: disable=no-member
+            elif isinstance(err, asyncio.TimeoutError):
+                err_log = str(traceback.format_exc())
+                msg = (
+                    "couldn't retrieve the necessary trust data for verification within 30s. "
+                    + "most likely there was a temporary network failure."
```

Comment:
you are guessing here. It maybe that the server cannot be reached. Would be a bit more instructive like "check connection..."